torch/quantization/quantize.py

```diff
@@ -80,6 +80,11 @@ def _observer_forward_pre_hook(self, input):
     # Returning nothing is Ok, Module._call_impl will intrepret this
     # as the pre_hook making no changes to the input, as desired
 
+def register_activation_post_process_hook(module):
```

Comment:
Can you explain on why this refactoring is needed?

Reply:
We need to use this function in module definition, see next PR

Reply:
Sorry, didnt see that before, its clear now.